packages/media/src/types/resource.ts

```diff
@@ -49,7 +49,7 @@ export interface ResourceSize {
   height: number;
 }
 
-export type ResourceId = string | number;
```

Comment:
This was `string | number` for a reason.

Media3p resources typically have string IDs, whereas WordPress media resources have numeric values.

Reply:
Sorry, a lot of the diff from here on down are due to the merge conflicts and not actually because I wanted to change what you had done, I just kept what I had. I'll go over everything here in more detail today.